docs/user/integrations.rst

```diff
@@ -1,17 +1,46 @@
-Connecting git repositories
-===========================
+..
+   Some points we want to cover in this article:
+   * Talk about the benefits of always up to date docs
+   * Discuss versioning in here, since it relies directly on Git?
+   * Have a small diagram that shows (You --push--> GitHub --webhook--> RTD --Build docs--> Deploy
+       (Perhaps reuse this: https://about.readthedocs.com/images/homepage.png)
+
+
+Building documentation on every git commit
+==========================================
 
 Read the Docs is a *Continuous Documentation Deployment* platform for your software project.
 Every time you change something in your documentation, Read the Docs will detect your change and build your documentation.
 This happens using *webhooks*.
 
-Integrations are configured with your repository provider,
+The Continuous Integration and Continuous Deployment (CI/CD) features are configured with your repository provider,
 such as GitHub, Bitbucket or GitLab,
-and with each change to your repository, Read the Docs is notified by the webhook.
+and with each change committed to your repository, Read the Docs is notified by the webhook.
 
 When we receive an integration notification, we determine if the change is related to an active version for your project.
 If it is, a build is triggered for that version.
 
+Continuous Documentation for software projects
+----------------------------------------------
+
+Documentation fits into any CI/CD pipeline through the idea and objectives of *Documentation as Code*.
+A straight-forward method to achieve this is by maintaining documentation code alongside the source code,
+meaning that the documentation's life-cycle is contained within the same git repository as the source code.
+
+When changes happen to the source code, changes should also happen to the documentation.
+By managing these changes in the same life-cycle,
+you can benefit from for instance both documentation and source code being subject of the same code reviews process.
```

Comment:
Haha, yes indeed! And really good to highlight this. Readers that are developers will get this message immediately.

I'm a bit worried that readers who aren't developers will have a harder time getting it. Here, the question becomes more fundamental than using Read the Docs.. why should you even store Documentation as Code with git? A quick way to solve it would be to add some external reference about this, in case we know a great one.

When you see a "for instance", it's because my texts are generated in a flow of thoughts that are "spoken" - I try to get them massaged into a precise and concise written form afterwards :)